classes.fields.php

```diff
@@ -223,7 +247,19 @@ public function get_the_name_attr( $append = null ) {
 	 */
 	public function class_attr( $classes = '' ) {
 
-		if ( $classes = implode( ' ', array_map( 'sanitize_html_class', array_filter( array_unique( explode( ' ', $classes . ' ' . $this->args['class'] ) ) ) ) ) ) { ?>
+		// Combine any passed-in classes and the ones defined in the arguments and sanitize them.
+		$all_classes = array_unique( explode( ' ', $classes . ' ' . $this->args['class'] ) );
+		$classes     = array_map( 'sanitize_html_class', $all_classes );
```

Comment:
The `array_filter` here might be useful to keep, as it removes any false-y values (empty string, e.g.)